DigitalOffice/Services/AuthenticationService/Startup.cs

```diff
@@ -0,0 +1,126 @@
+using LT.DigitalOffice.AuthenticationService.Validators;
+using LT.DigitalOffice.AuthenticationService.Token;
+using LT.DigitalOffice.AuthenticationService.Broker.Senders;
+using LT.DigitalOffice.AuthenticationService.Broker.Senders.Interface;
+using LT.DigitalOffice.AuthenticationService.Token.Interfaces;
+using LT.DigitalOffice.AuthenticationService.Commands.Interfaces;
+using LT.DigitalOffice.Broker.Responses;
+
+namespace LT.DigitalOffice.AuthenticationService
+{
+    public class Startup
+    {
+        public IConfiguration Configuration { get; }
+
+        public Startup(IConfiguration configuration)
+        {
+            Configuration = configuration;
+        }
+
+        public void ConfigureServices(IServiceCollection services)
+        {
+            ConfigJwt(services);
+
+            services.AddHealthChecks();
+
+            services.AddControllers();
+
+            ConfigRabbitMq(services);
+
+            services.AddMassTransitHostedService();
+
+            ConfigBrokerSenders(services);
+            ConfigCommands(services);
+            ConfigValidators(services);
+        }
+
+        private void ConfigJwt(IServiceCollection services)
+        {
+            var signingKey = new SigningSymmetricKey();
+            var signingDecodingKey = (IJwtSigningDecodingKey)signingKey;
+
+            services.AddSingleton<IJwtSigningEncodingKey>(signingKey);
+            services.AddSingleton<NewToken>();
+
+            services.Configure<TokenOptions>(Configuration.GetSection("TokenSettings"));
+
+            services.AddAuthentication(JwtBearerDefaults.AuthenticationScheme)
+            .AddJwtBearer(options =>
+            {
+                options.RequireHttpsMetadata = true;
+                options.TokenValidationParameters = new TokenValidationParameters
+                {
+                    ValidateIssuer = true,
+                    ValidIssuer = Configuration.GetSection("TokenSettings:TokenIssuer").Value,
+                    ValidateAudience = true,
+                    ValidAudience = Configuration.GetSection("TokenSettings:TokenAudience").Value,
+                    ValidateLifetime = true,
+                    IssuerSigningKey = signingDecodingKey.GetKey(),
+                    ValidateIssuerSigningKey = true
+                };
+            });
+        }
+
+        private void ConfigRabbitMq(IServiceCollection services)
+        {
+            string appSettingSection = "ServiceInfo";
+            string serviceId = Configuration.GetSection(appSettingSection)["ID"];
+            string serviceName = Configuration.GetSection(appSettingSection)["Name"];
+
+            var uri = $"rabbitmq://localhost/UserService_{serviceName}";
+
+            services.AddMassTransit(x =>
+            {
+                x.UsingRabbitMq((context, cfg) =>
+                {
+                    cfg.Host("localhost", "/", host =>
+                    {
+                        host.Username(serviceName);
```

Comment:
I think, that username should contain serviceId, because there can be several services with the same name.

Reply:
Can add in the appsetting.json this prefix "_1(or any other number)" (serviceName_1)